src/functions/Output.ps1

```diff
@@ -577,9 +577,12 @@ function Get-WriteScreenPlugin ($Verbosity) {
 
             Skipped {
                 if ($PesterPreference.Output.Verbosity.Value -in 'Normal', 'Detailed', 'Diagnostic') {
-                    $because = if ($_test.FailureMessage) { ", because $($_test.FailureMessage)" } else { $null }
                     & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Skipped "$margin[!] $out" -NoNewLine
-                    & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Skipped ", is skipped$because" -NoNewLine
+                    & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Skipped " is skipped" -NoNewLine
```

Comment:
I think you misunderstood me, what I meant by another layer was that this code should only look at FailureMessage, and FailureMessage is imho set in Invoke-TestItem. And in here we should only take the message and print it, if possible. But that is not a huge problem, I can fix that later in 5.1. 

What is problematic is that the Data hashtable is for TestCases data, you should not modify that. Please just revert this to the previous solution, to not make this change unnecessarily difficult.

Reply:
OK, I fixed this by overloading your CreateErrorRecord to allow me to specify the error type for PesterTestSkipped (which you specified elsewhere in the code in a -contains but it really should be an enum)